### PR TITLE
[10.0] add new addon to allow to display the category image if the product does not have one

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**

### CODE REVIEW

File: shopinvader_category_image_for_product/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Shopinvader category image for product",
+    "summary": "Shopinvader Display category image for product",
+    "version": "10.0.1.0.0",
+    "category": "e-commerce",
+    "author": "ACSONE SA/NV",
+    "website": "http://www.shopinvader.com",
```
Comment (simahawk) [inline shopinvader_category_image_for_product/__manifest__.py]: https://github.com/shopinvader/odoo-shopinvader

File: shopinvader_category_image_for_product/models/shopinvader_variant.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    )
+    def _compute_image_categories(self):
+        """
+        Compute function for image_categories field.
+        This field should contains the value of images field into the first
+        category found (with same backend)
+        :return:
+        """
+        for record in self:
+            category = record.shopinvader_categ_ids.filtered(
```
Comment (simahawk) [inline shopinvader_category_image_for_product/models/shopinvader_variant.py]: can you split this to `_get_category_by_backend`? or similar?

File: shopinvader_category_image_for_product/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+This paddon allows to display the category image on a product without images
```
Comment (simahawk) [inline shopinvader_category_image_for_product/readme/DESCRIPTION.rst]:
```suggestion
This addon allows to display the category image when a product has no images.
```

**Review by simahawk — CHANGES_REQUESTED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by Cedric-Pigeon** [comment]
@simahawk Are you still waiting something from me on this one?

**Review by simahawk — APPROVED** [review]

**Review by sebastienbeau — APPROVED** [review]

**Pull request merged**